pkg/gui/controllers/helpers/confirmation_helper.go

```diff
@@ -334,6 +334,12 @@ func (self *ConfirmationHelper) resizeMenu() {
 	selectedItem := self.c.Contexts().Menu.GetSelected()
 	if selectedItem != nil {
 		tooltip = selectedItem.Tooltip
+		if selectedItem.DisabledReason != "" {
+			if tooltip != "" {
+				tooltip = tooltip + "\n\n"
+			}
+			tooltip = tooltip + "Disabled: " + selectedItem.DisabledReason
+		}
```

Comment:
This should go into a helper function, I just couldn't decide where to put it. We need it from here and from `MenuController.GetOnFocus`.

Reply:
I decided to (somewhat randomly) put it in `ConfirmationHelper` (a9f16ffd535a). Also localized the "Disabled:" string.